clients/drcachesim/reader/reader.cpp

```diff
@@ -174,6 +174,7 @@ reader_t::process_input_entry()
             cur_ref_.instr.tid = cur_tid_;
             cur_ref_.instr.type = (trace_type_t)input_entry_->type;
             cur_ref_.instr.size = input_entry_->size;
+            cur_ref_.instr.indirect_branch_target = last_branch_target_;
```

Comment:
We always set indirect_branch_target? I don't see last_branch_target_ being set to zero beyond init. Are we just letting it be a non-zero val and expecting users to not use it if it's not one of the indirect branch types? If yes, it may cause some confusion.

Reply:
Good point.  Added a check for an indirect branch type; if it's not, this field is set to 0.